### PR TITLE
Doc updates for can-symbol symbols.

### PR DESCRIPTION
**Pull request by bmomberger-bitovi**

### CODE REVIEW

File: docs/symbols-getKeyDependencies.md [hunk 1]
```diff
@@ -3,4 +3,49 @@
...
+  		}
+  	});
+  }
+  return ret;
+};
+```
+
+@this {Object} obj a Map-like object
+@param {String} key a property key on `this`
+@return {Object} an object of key dependencies in a CIDMap and value dependencies in a CIDSet
```
Comment (justinbmeyer) [inline docs/symbols-getKeyDependencies.md]: this should probably specify the keys in the object like:

`{{keyDependencies: CIDMAP, valueDependencies: CIDSet}}`

Comment (bmomberger-bitovi) [inline docs/symbols-getKeyDependencies.md]: It is clearer in the description (the keys used are described by name).  If you'd like it in the return spec as well, please provide an example of the format you'd like to see there, b/c I don't think I've seen anything like it in the other areas of the docs.

Comment (justinbmeyer) [inline docs/symbols-getKeyDependencies.md]: `{{keyDependencies: CIDMAP, valueDependencies: CIDSet}` is the format.  Unless I am missing something.

**Comment by phillipskevin** [comment]
@bmomberger-bitovi any reason we want to continue to hide the docs for `getKeyDependencies` / `getValueDependencies`?

**Comment by bmomberger-bitovi** [comment]
No, the `@hide` tag should have been removed from `getValueDependencies` (it's already not in `getKeyDependencies`).

**Pull request merged**